docs/sources/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -79,6 +79,14 @@ Using a custom namespace solves problems later on because you do not have to ove
          # empty, disabled.
    ```
 
+   > **Note:** To see all of the configurable parameters for a Helm chart installation, use `helm show values grafana/mimir-distributed`.
+   > If you want to get the values file from Github, build the URL as follows because we git tag every release:
+   > `https://github.com/grafana/mimir/blob/mimir-distributed-<chart-version>/operations/helm/charts/mimir-distributed/values.yaml`.
+   > For example, `https://github.com/grafana/mimir/blob/mimir-distributed-3.1.0/operations/helm/charts/mimir-distributed/values.yaml`.
+   > You must use a matching Helm chart version, because charts are not guaranteed to be backwards compatible between major version.
+   > You have to merge every change from the new version to keep it
+   > working and not just what you actually customized.
```

Comment:
I'm not sure I understand this bit. Are we asking them to pull in all changes between values files versions regardless of whether they customized it or now?

So if I had in my values file

```yaml
global:
  podAnnotations:
    x: y
```
and version 4.7.0 of the chart introduced changes to, e.g.

```yaml
nginx:
  someNewOption: 123
```

then they should add this new nginx field to their values file? This doesn't sound right

Reply:
Come to this after a discussion with Krajo. However, now I feel it is not right. Definitely, it shouldn't be the scenario above. I will remote this last sentence for now.